lib/pure/complex.nim

```diff
@@ -376,6 +376,15 @@ proc `$`*(z: Complex): string =
 
 {.pop.}
 
+proc FFT*(dataSource : seq[float]):seq[Complex]= 
```

Comment:
Also, please add documentation for this procedure. What is ``fft``? What does it stand for and what does it do?